src/components/ArticleCard.jsx

```diff
@@ -12,17 +13,53 @@ export default function ArticleCard({
   comment_count,
   setArticleToRead,
   body,
+  setUpdatedVote,
+  setUserVote,
+  votingError,
 }) {
-  // function formatDate(date) {
-  //   return new Date(date).toDateString();
-  // }
+  const [hasVoted, setHasVoted] = useState(false);
+  const [upvoteButtonStatus, setUpvoteButtonSatus] = useState(false);
+  const [downvoteButtonStatus, setDownvoteButtonStatus] = useState(false);
 
   function readArticle() {
     setArticleToRead(article_id);
   }
 
+  function handleUpvote() {
```

Comment:
Could we make this portion of code a bit more DRY? We have two functions doing more a less the same thing with one minor difference, +1 or -1. If we can get that value from the html tag that has been clicked instead we could combine these functions nicely.